Sources/TuistDependencies/Carthage/CarthageInteractor.swift

```diff
@@ -175,24 +161,22 @@ public final class CarthageInteractor: CarthageInteracting {
     /// Saves lockfile resolved dependencies in `Tuist/Dependencies` directory.
     private func saveDependencies(pathsProvider: CarthagePathsProvider) throws {
         // validation
```

Comment:
Nitpick: I'd remove the comments that are redundant with the method calls. If the method name is explicit enough to tell the reader of the code what the method does, the comments are redundant.